### ISSUE TITLE
SCRIPT ERROR: @esx_illegal/server/chemicals.lua:7: attempt to call a nil value (field 'canCarryItem')

### ISSUE THREAD

**Issue by Evilcho0TM** [Xovos/esx_illegal#23]
How can i manage it

**Comment by Xovos** [comment]
[https://github.com/Xovos/esx_illegal/tree/pre-limit-update](https://github.com/Xovos/esx_illegal/tree/pre-limit-update)

**Comment by Evilcho0TM** [comment]
I use tihs version

**Comment by Xovos** [comment]
Update your ESX or use the other version.

**Comment by miraz559** [comment]
how can i fix this error https://prnt.sc/s87bcb

**Comment by miraz559** [comment]
its showing attempt to call nill value (field 'canSwapItme')

**Comment by Xovos** [comment]
Hi

Please use this version of the script: [https://github.com/Xovos/esx_illegal/tree/pre-limit-update](https://github.com/Xovos/esx_illegal/tree/pre-limit-update)

**Comment by miraz559** [comment]
still not working the error is still there https://prnt.sc/s87qg7

**Comment by Xovos** [comment]
Hi

Did you also re-add the sql script?

**Comment by miraz559** [comment]
yes

**Comment by miraz559** [comment]
i tried in many way plus i tried changing sql file and es version but it is the same

**Comment by miraz559** [comment]
but the esx drug works good but the esx illegal doesn't work

**Comment by Xovos** [comment]
Hi

You are not the first with this problem and others have gotten it fixed this way [https://github.com/Xovos/esx_illegal/issues/22#issuecomment-619425381](https://github.com/Xovos/esx_illegal/issues/22#issuecomment-619425381)

**Comment by miraz559** [comment]
ok let me try again

**Comment by miraz559** [comment]
i tried again with new sql file also but result is the same https://prnt.sc/s8ukpa still showing the problem it doesn't show any option to pickup when i try to pickup something this error shows up

**Comment by miraz559** [comment]
can you please tell me how can i fix this error

**Comment by Xovos** [comment]
Hi

I don't know why it's giving that error it should work, it worked for other people and I'm sorry, but I currently don't have the time to look into it.
You could try the master branch once again, maybe it works this time.

**Comment by miraz559** [comment]
can you please tell me which ESX version is comparable with this script

**Comment by Xovos** [comment]
The latest
